protractor/e2e/integretion/second-page.spec.ts

```diff
@@ -55,9 +57,22 @@ describe('test for second page', () => {
         await secondPage.waitForElementVisible(secondPage.pdfContent);
     });
 
-    fit('info card error case', async () => {
+    it('info card error case', async () => {
         await secondPage.scrollToElement(secondPage.infoCardSection);
         await secondPage.waitForElementVisible(secondPage.infoEmailInpt);
         await secondPage.infoSubmitBtn.click();
     });
+
+    fit('should open page blog pages with articles', async () => {
+        await secondPage.waitForElementVisible(secondPage.blogHeaderLink);
+        await secondPage.blogHeaderLink.click();
+        await blogPage.waitForElementVisible(blogPage.postClass.first());
+        expect(await blogPage.getCurrentUrl()).toContain(blogPage.pageUrl);
+        expect(await blogPage.postClass.count()).toEqual(20);
+        await blogPage.scrollToElement(blogPage.paginationBlock);
+        await blogPage.secondBlogPageLink.click();
+        await secondPage.waitForElementVisible(blogPage.postClass.first());
+        expect(await blogPage.getCurrentUrl()).toContain('page/2');
+        expect(await blogPage.postClass.count()).toBeGreaterThan(1);
```

Comment:
7. Then second page opens and there is as minimum 1 article (page is not empty) - this is your task
as minimum 1 means, that your count should be greater than 0. Just imagine, if 2d page will have only 1 article, your test will fail, but should not.
expect(await $$('article header').count()).not.toBeNull();

Reply:
agree

Reply:
But count shouldn't return Null, we should use toBeGreaterThan(0) assert